### PR TITLE
DELIA-60903: Remote Control plugin - add findMyRemote method and irPr…

### PR DESCRIPTION
**Pull request by egalla204**
add findMyRemote method and IR programming state to status structure
Here is the link to internal Comcast gerrit review which passed verification builds:
https://gerrit.teamccp.com/#/c/736643/

### CODE REVIEW

**Review by cbucht200 — COMMENTED** [review]
Looks good, Idk if you are supposed to generate the MD file or not..

**Review by cbucht200 — APPROVED** [review]
Looks good to me

File: RemoteControl/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+    * **Security** in case of vulnerabilities.
+* Changes in CHANGELOG should be updated when commits are added to the main or release branches. There should be one CHANGELOG entry per JIRA Ticket. This is not enforced on sprint branches since there could be multiple changes for the same JIRA ticket during development. 
 
-All notable changes to this project will be documented in this file.
+* For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
+* The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [1.0.1] - 2023-04-12
```
Comment (apatel859) [inline RemoteControl/CHANGELOG.md:16]: CAn we bump version to 1.1.0 as minor change only applicable when minor bug fixes not not adding new API. Hope "Fixed netTypesSupported to be an array and not a string" change is backword compatible.

File: RemoteControl/RemoteControl.cpp [hunk 2]
```diff
@@ -1276,6 +1277,64 @@ namespace WPEFramework {
...
+                    returnResponse(false);
+                }
+                netType = value;
+                LOGINFO("netType passed in is <%s>.", networkTypeStr(netType));
+            }
+
+            std::string levelStr;
+            ctrlm_fmr_alarm_level_t level;
+            paramKey = "level";
+            if (parameters.HasLabel(paramKey))
```
Comment (apatel859) [inline RemoteControl/RemoteControl.cpp:1315]: This condition can be combined with line 1289

**Review by apatel859 — CHANGES_REQUESTED** [review]
MD

**Review by apatel859 — APPROVED** [review]

**Pull request merged**